src/base/static/js/views/app-view.js

```diff
@@ -293,7 +295,7 @@ export default Backbone.View.extend({
     // that's currently set in the address search bar.
     $(Shareabouts).on("mapdragend", function(evt) {
       if (self.isAddingPlace()) {
-        self.conditionallyReverseGeocode();
+        // no-op
```

Comment:
`conditionallyReverseGeocode` was a no-op, and wasn't being used elsewhere, so I deleted it.

Ideally, at some point, we can clean up this logic a bit more.